### DISCUSSION TITLE
Manual/class-MonoManager

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#3] [Notes]
# Manual/class-MonoManager

https://docs.unity3d.com/Manual/class-MonoManager.html

**Comment by vertxxyz** [comment]
Use the [`DefaultExecutionOrder`](https://docs.unity3d.com/ScriptReference/DefaultExecutionOrder.html) attribute to define script execution order via a class-level attribute.

**Comment by vertxxyz** [comment]
You can't override the execution order of scripts in packages (as the order is serialized to the script's meta file).
